cmd/searchlight/run.go

```diff
@@ -53,19 +54,18 @@ func NewCmdRun() *cobra.Command {
 
 			kubeClient = clientset.NewForConfigOrDie(config)
 			extClient = tcs.NewForConfigOrDie(config)
+			if icingaSecretName == "" {
```

Comment:
There is one problem with this. If the sidecar icinga has not started yet, this results in crashed operator container and eventual crashloop. We need a way to block for this, if we open connection in the beginning. Also, we need to be able to reconnect incase icinga pod crashed and restarted.